src/main/interface/settings.c

```diff
@@ -274,6 +274,12 @@ static const char * const lookupTableRatesType[] = {
     "BETAFLIGHT", "RACEFLIGHT"
 };
 
+#ifdef USE_GYRO_EXPERIMENTAL_FILTER
```

Comment:
I suggest using `USE_GYRO_STAGE2_FILTER` for this define (and updating all places that refer to it)